modules/programs/neovim.nix

```
@@ -55,6 +55,14 @@ in
         '';
       };
 
+      vimdiffAlias = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Alias `vimdiff` to `nvim -d`.
+        '';
+      };
```

I would suggest changing the option name to a more explicit one. For example
```suggestion
      enableVimdiffShellAlias = mkEnableOption "" // {
        description = ''
          Whether to bind shell alias <command>vimdiff</command>
          to <command>nvim -d</command>.
        '';
      };
```

---

it's to keep consistent with the other commands, they are named smilarily: `viAlias` and `vimAlias`

---

Ah, of course. I didn't pay enough attention 😬